lib/rules/declaration-empty-line-before/__tests__/index.js

```diff
@@ -32,46 +33,49 @@ testRule(rule, {
 
   reject: [ {
     code: "a { top: 15px; }",
+    fixed: "a {\n\n top: 15px; }",
```

Comment:
We have rules designed to do only their thing. We don't have overlapping rules. So why *-empty-line-before should care about anything except empty lines before? :) We have `indentation` rule to take care of indentation.